19.md

```diff
@@ -35,18 +35,23 @@ These are the possible bech32 prefixes with `TLV`:
   - `nprofile`: a nostr profile
   - `nevent`: a nostr event
   - `nrelay`: a nostr relay
+  - `nitem`: a nostr parameterized replaceable event coordinate (NIP-33)
 
 These possible standardized `TLV` types are indicated here:
 
 - `0`: `special`
   - depends on the bech32 prefix:
     - for `nprofile` it will be the 32 bytes of the profile public key
     - for `nevent` it will be the 32 bytes of the event id
-    - for `nrelay`, this is the relay URL.
+    - for `nrelay`, this is the relay URL
+    - for `nitem`, it is the identifier (the `"d"` tag) of the event being referenced
```

Comment:
I think kind is needed here no? or am I missing something? 

why not make 'nitem' applicable to general replaceable events as well? or is that bad form? first '0' type would be kind and second '0' type would be d-identifier.

Reply:
Yes, I think it is applicable, we just don't have a use case for that because generally the kind is implied by the purpose already so we only need the public key.

Reply:
Added this to the `nitem` encoding.